sim_pipeline/gg_lens_table.py

```diff
@@ -0,0 +1,65 @@
+from astropy.table import Table
+from sim_pipeline.Pipelines.skypy_pipeline import SkyPyPipeline
+from sim_pipeline.gg_lens import GGLens, theta_e_when_source_infinity
+import numpy as np
+
+
+def create_table(
+    gg_lens_population,
+    parameters=[
+        "einstein_radius",
+        "ellipticity",
+        "velocity_dispersion",
+        "stellar_mass",
+        "magnitude",
+        "source_redshift",
+        "lens_redshift",
+    ],
+):
+    """Creates an astropy table for a given gg_lens population.
+
+    :param gg_lens_population: List of GGLens instances.
+    :type gg_lens_population: list
+    :param parameters: List of parameters to extract from the population.
```

Comment:
this deserves a bit more explanation. These are de-facto definitions in the gg_lens class